marlowe/pab/Main.hs

```diff
@@ -36,19 +40,12 @@ import qualified Plutus.PAB.Simulator                as Simulator
 import           Plutus.PAB.Types                    (PABError (..))
 import qualified Plutus.PAB.Webserver.Server         as PAB.Server
 import qualified PlutusTx.AssocMap                   as AssocMap
-import           Wallet.Emulator.Types               (Wallet (..))
+import           Text.Read                           (readMaybe)
 
 main :: IO ()
 main = void $ Simulator.runSimulationWith handlers $ do
     Simulator.logString @(Builtin Marlowe) "Starting marlowe PAB webserver on port 8080. Press enter to exit."
     shutdown <- PAB.Server.startServerDebug
```

Comment:
Not related to this PR, but while we're here... Is there a reason for using `startServerDebug` instead of `startServer`? This ties us into port 8080, which makes things a bit awkward for frontend development (where the regular PAB is set up to listen on 9080). It would be easier for me to have them on the same port, so I can easily switch between the two.

Reply:
In fact, now that I think of it, if the `marlowe-pab` is to be usable with Marlowe Run, we'll need it to accept some custom config - the default settings won't do.

Reply:
No reason other than that the simulator doesn't yet read the config (but we will make that change soon)